.travis.yml

```diff
@@ -1,21 +1,38 @@
-language: python
 sudo: false
-python:
-  - 2.7
-  - 3.5
-  - 3.6
-env:
-  - DJANGO_VERSION=1.8.*
-  - DJANGO_VERSION=1.9.*
-  - DJANGO_VERSION=1.10.*
-  - DJANGO_VERSION=1.11.*
+dist: trusty
+language: python
+cache: pip
+matrix:
+  fast_finish: true
+  include:
+  - env: TOXENV=py27-dj111
+    python: 2.7
+  - env: TOXENV=py34-dj111
+    python: 3.4
```

Comment:
Why do you add python-3.4 testing?
django-redshift-backend doesn't support python-3.4.